pepper-apis/config/testing-inmemorydb.conf.ctmpl

```diff
@@ -7,7 +7,8 @@
 {{with $testingConf := vault (printf "secret/pepper/%s/%s/testing" $environment $version)}}
 {{with $auth0 := index $testingConf.Data "auth0"}}
 {{with $testingAuth := index $testingConf.Data "auth0"}}
-{{with $elasticsearchConf := vault (printf "secret/pepper/%s/%s/elasticsearch" $environment $version)}}
+{{with $conf := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
+{{with $fileUploads := index $conf.Data "fileUploads"}}
```

Comment:
For some reason, config rendering was failing for this file, so I had to do this `index` thing. Not sure why.